nixos/modules/services/continuous-integration/buildkite-agent.nix

```diff
@@ -62,16 +56,20 @@ in
       };
 
       openssh =
-        { privateKey = mkOption {
-            type = types.str;
+        { privateKeyPath = mkOption {
```

Comment:
All three option renames are missing `mkRenamedOptionModule` (grep for examples)

Reply:
Done.